src/power_divergence.jl

```diff
@@ -67,18 +67,18 @@ one of the following methods. Possible values for `method` are:
   * Gold, R. Z. Tests Auxiliary to ``χ^2`` Tests in a Markov Chain. Annals of
     Mathematical Statistics, 30:56-74, 1963.
 """
-function StatsBase.confint(x::PowerDivergenceTest; level::Float64=0.95,
-                           tail::Symbol=:both, method::Symbol=:auto, correct::Bool=true,
-                           bootstrap_iters::Int64=10000, GC::Bool=true)
+function StatsAPI.confint(x::PowerDivergenceTest; level::Float64=0.95,
+                 tail::Symbol=:both, method::Symbol=:auto, correct::Bool=true,
+                 bootstrap_iters::Int64=10000, GC::Bool=true)
```

Comment:
```suggestion
function StatsAPI.confint(x::PowerDivergenceTest; level::Float64=0.95,
                          tail::Symbol=:both, method::Symbol=:auto, correct::Bool=true,
                          bootstrap_iters::Int64=10000, GC::Bool=true)
```
Looks like alignment got messed up somehow